frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.scss

```diff
@@ -0,0 +1,12 @@
+.co-guided-tour-drawer {
+  &__panel-content {
+    overflow-y: auto !important;
+    z-index: auto;
+  }
+
+  &__subtitle {
+    display: inline;
+    padding-left: var(--pf-global--spacer--md);
```

Comment:
Padding should be moved to the right side of the title so that if the name is so long that the subtitle needs to wrap to the next line, it won't have an awkward left indent.

Reply:
done